docs/user-guide.rst

```diff
@@ -417,6 +418,7 @@ Common build options
    extension, they are always saved if they are detected regardless of the value
    of this option. If enabled, it is needed to use a compiler that supports the
    option ``-msign-return-address``. It defaults to 0.
+   This is an experimental feature.
```

Comment:
Suggest: "Boolean option to enable ARMv8.3 Pointer Authentication
(``ARMv8.3-PAuth``) support for TF-A BL images itself. If enabled, it is needed to use a compiler that supports the
option ``-msign-return-address``. It defaults to 0 and this is an experimental feature.  Note that Pointer Authentication is enabled in Non-Secure world irrespective of any build flag if the CPU supports it."